conda_recipe/conda_build_config.yaml

```diff
@@ -31,9 +31,9 @@ pin_run_as_build:
   nlohmann_json:
     max_pin: x
   numba:
-    max_pin: x
+    max_pin: x.x
```

Comment:
I think this is reasonable. But I'm curious if there is a specific reason to increase this pinning.